website/project/model.py

```diff
@@ -2728,6 +2728,18 @@ def forked_from_id(self):
         return None
 
     @property
+    def templated_from(self):
```

Comment:
As mentioned above, I don't _think_ you need to change the model at all to make this work. You should be able to drop the changes to this file.